src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -333,13 +333,28 @@ message SampleValueSet {
   //   several placeholders.
   repeated string parameters = 4;
 
-  repeated string sample_args = 5;
+  // Attributes governing how to init request object.
+  // Each attribute must have unique, non-overlapping, path.
+  repeated SampleInitAttribute attributes = 5;
```

Comment:
As per our discussion, `SampleValueSet` should have its field `parameters` contain two parts: a `repeated string` of default key/value pairs, and a `repeated SampleInitAttribute` for their metadata. Do you have a reason for wanting to change that?

```
sample_value_set:
   …
  parameters:
    defaults:
      - alice=bob
      - carl=denise
      - …
    attributes:
      parameter: carl
      sample_argument: true
```